scripts/generate-ma-zips.js

```diff
@@ -0,0 +1,39 @@
+// Generate a pared down list of Massachusetts zipcodes with their
+// longitude and latitude.
+
+const fs = require("fs");
+const zipcodes = require("zipcodes-perogi");
+
+const MAZipcodes = zipcodes.lookupByState("MA");
+
+// 'zipcodes-perogi' gives us this form, which is more than we need:
```

Comment:
suuuuper nit but could you use /* instead of the individual line comments?

Reply:
Sure, but…why? What's the rule? If that's a project style, shouldn't prettier enforce it?

I haven't heard anyone argue for mulitline /* comments in ages, but I've definitely heard good arguments for // (namely, ease of using simple editor tools to insert and remove them, and not having to deal with the computational complexity of nesting if you block-comment-out a region or worse, reverse it.)

Reply:
Hm, I don't think prettier has a setting for it. But I don't care enough so if you're hesitant to change it we can leave it. I like it for readability personally - lots of `//` lines feels cluttered to me 🤷

Reply:
> if you're hesitant to change it we can leave it.

Nah, it's changed: 6f3f931c6bac439f4e5d19a9dfab0e8e1c959002

I just like to know why: I always assume I was one of those kids who went around asking "Why is the sky blue?" long past "Rayleigh scattering" and so I do want to know what the thinking is. (Sounds like the answer is "mental model of neatness that is formed by habit but not grounded in clearly-articulated principles," which is fine.)

Of course, as a philosophical matter, style opinions that are not written down and not enforced by the code formatter are maybe not the best. And where they seem to differ with others considered defaults (but not preferences), that's extra friction.

While I'm here, I force-pushed over your merge commit (2a00ba5d637bd268d3247e8fe8070e9054385133), I don't think it's necessary to have in here and think it will either lead to a less clean merge later or I'll want to rebase the branch before merging. (But this PR is definitely a candidate for a squash merge, if not for the prettier stuff, so cleanliness of branch history isn't a big deal.) I can bring it back if you want tho.